Classes/Domain/Model/Forum/Attachment.php

```diff
@@ -218,4 +221,24 @@ public function getDownloadCount() {
 	public function increaseDownloadCount() {
 		$this->downloadCount++;
 	}
+
+    /**
+     * Gets the whole TCA config of tx_typo3forum_domain_model_forum_attachment
+     * @return array The whole TCA config of tx_typo3forum_domain_model_forum_attachment
+     */
+    public function getTCAConfig() {
```

Comment:
Oh, see it now. If this method didn't exist until now, how can this have _ever_ worked at all?